src/experimental/build/commands/compile/javascript.ts

```diff
@@ -49,7 +55,7 @@ export function getPrelude(main: string): string {
 
         import { getDefaultVisitorData, IDL, idlToString } from 'azle';
         export { Principal } from '@dfinity/principal';
-        import * as Canister from './${main}';
+        import * as Canister from '${main}';
```

Comment:
We shouldn't expose the absolute path in the main.js

Reply:
done